packages/terra-overlay/src/terra-dev-site/doc/example/OverlayExample.jsx

```diff
@@ -33,8 +33,10 @@ class OverlayExample extends React.Component {
         <Overlay isOpen={this.state.show} isRelativeToContainer={this.state.relative} onRequestClose={this.handleOnRequestESC} zIndex="6000">
           <p>Close by clicking the overlay or pressing the ESC key.</p>
         </Overlay>
-        <Button text="Trigger Container Overlay" onClick={this.handleTriggerOverlay} />
-        <Button text="Trigger FullScreen Overlay" onClick={this.handleTriggerFullScreenOverlay} />
+        <div>
```

Comment:
What is the div needed for?

Reply:
I added This div wrapper to group the elements which are not part of overlay. By doing this the number elements that will be iterated in for loop of method [disableContainerChildrenFocus()](disableContainerChildrenFocus) will be reduced since container will have only one child element div to loop. 
without wrapper div method will loop through all the elements to set attribute aria-hidden property.  
with @emilyrohrbough  comment got to know div should be added around children prop at OverlayContainer itself instead of adding it on consuming code hence will be removing div from here.